spec/unit/shoulda/matchers/action_controller/callback_matcher_spec.rb

```diff
@@ -2,6 +2,92 @@
 
 describe Shoulda::Matchers::ActionController::CallbackMatcher, type: :controller do
   shared_examples 'CallbackMatcher' do |kind, callback_type|
+    shared_examples 'qualified with conditional' do |conditional|
+      context "qualified with #{conditional}" do
+        let(:action) { :index }
+        let(:multiple_actions) { [:index, :show] }
+        let(:other_action) { :other_action }
+
+        def match_with_conditional(conditional, params)
+          match.send(conditional, params)
+        end
+
+        context "and when a #{kind} hook is in place with the qualifier" do
+          it "accepts" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.